src/taskEvents/utils.js

```diff
@@ -0,0 +1,124 @@
+const BASE_URL = 'https://api.realdevsquad.com';
+
+function createElement({ type, attributes = {}, innerText }) {
+  const element = document.createElement(type);
+  Object.keys(attributes).forEach((item) => {
+    element.setAttribute(item, attributes[item]);
+  });
+  element.textContent = innerText;
+  return element;
+}
+
+function addLoader(container) {
+  const loader = createElement({
+    type: 'div',
+    attributes: { class: 'loader' },
+  });
+  const loaderP = createElement({
```

Comment:
What does loaderP stand for?
Please rename it

Reply:
okay I will rename it